core/src/com/unciv/logic/map/TileInfo.kt

```diff
@@ -574,7 +568,7 @@ open class TileInfo {
             "Water resource" -> isWater && observingCiv != null && hasViewableResource(observingCiv)
             "Natural Wonder" -> naturalWonder != null
             "Featureless" -> terrainFeatures.isEmpty()
-            "Fresh Water" -> isAdjacentToFreshwater
+            "Fresh Water" -> isAdjacentTo(Constants.freshWater)
```

Comment:
So we still have both "Fresh water" and "Fresh Water" and support only one or both depending on the unique?

Reply:
How about `const`ing that one as well, then naming one freshWaterSource and the other freshWaterAccess (the latter the one with capital Water), irrespective of actual string value - so the uses are traceable and changing the actual keys, if at all, becomes more readable?

Reply:
The goal is to remove all support for "Fresh Water", but this will I think require adding a warning in the ruleset checks